src/router.js

```diff
@@ -21,6 +22,11 @@ export default new Router({
       // which is lazy-loaded when the route is visited.
       component: () =>
         import(/* webpackChunkName: "about" */ "./views/About.vue")
+    },
+    {
+      path: "/login",
+      name: "login",
+      component: Login
```

Comment:
We will need a LoginView here, so please remove this from the PR, we can just focus on the Login Component only.

Reply:
removed it